src/client/java/teammates/client/scripts/sql/DataMigrationEntitiesBaseScriptSql.java

```diff
@@ -46,11 +46,6 @@ public abstract class DataMigrationEntitiesBaseScriptSql<
     // the folder where the cursor position and console output is saved as a file
     private static final String BASE_LOG_URI = "src/client/java/teammates/client/scripts/log/";
 
-    // 100 is the optimal batch size as there won't be too much time interval
-    // between read and save (if transaction is not used)
-    // cannot set number greater than 300
-    // see
-    // https://stackoverflow.com/questions/41499505/objectify-queries-setting-limit-above-300-does-not-work
```

Comment:
I've verified that this is no longer true. Running it on staging with 1000 does not lead to any crashes. However, it's slower on average compared to the original 100.